src/shaders/example/fragment.glsl

```diff
@@ -0,0 +1,5 @@
+#version 120
```

Comment:
I'm not sure if you ran into this problem as well with your Angular Three.js starter kit, but when I try adding these fragment and vertex shaders to a `ShaderMaterial` and using it in the scene, I get the following message during runtime:

```
THREE.WebGLProgram: Shader Error 0 - VALIDATE_STATUS false

Program Info Log: Vertex shader is not compiled.
 
VERTEX

ERROR: 0:57: 'version' : #version directive must occur before anything else, except for comments and white space
 

  52: #ifdef USE_SKINNING
  53: 	attribute vec4 skinIndex;
  54: 	attribute vec4 skinWeight;
  55: #endif
  56: 
> 57: #version 120
  58: 
  59: void main() {
  60:     gl_Position = projectionMatrix * viewMatrix * modelMatrix * vec4(position, 1.0);
  61: }
FRAGMENT

ERROR: 0:31: 'version' : #version directive must occur before anything else, except for comments and white space
 

  26: vec4 LinearTosRGB( in vec4 value ) {
  27: 	return vec4( mix( pow( value.rgb, vec3( 0.41666 ) ) * 1.055 - vec3( 0.055 ), value.rgb * 12.92, vec3( lessThanEqual( value.rgb, vec3( 0.0031308 ) ) ) ), value.a );
  28: }
  29: vec4 linearToOutputTexel( vec4 value ) { return LinearToLinear( value ); }
  30: 
> 31: #version 120
  32: 
  33: void main() {
  34:     gl_FragColor = vec4(1.0, 1.0, 1.0, 1.0);
  35: }
```

It looks like other directives get added before the version. Without the version directive, the shaders run fine. 🤔

Reply:
Oh yes, it's my fault, it's the default generation of my IDE.
ShaderMaterial adds a bunch of stuff before your glsl shader (boilerplate, default uniforms etc etc).
It's not the case if you use RawShaderMaterial, which is the most primary form of GLSL shader. 

I just tested that the import was OK, and didn't not pay too much attention about the "valid" status of my shader file. 
We can just remove the #version and it would work.

Reply:
Perfect! 😀